client/app/login/login.js

```diff
@@ -1,4 +1,8 @@
 angular.module('jobTracker.login', [])
-.controller('loginController', function($scope, AuthFactory) {
-
+.controller('loginController', function($scope, AuthFactory, $location) {
+	$scope.login = function () {
+		AuthFactory.login($scope.user.username, $scope.user.password).then((data)=> {
```

Comment:
make the change you suggested about passing the user object, make sure to do that in the factory as well